src/bundle/Resources/config/services.yml

```diff
@@ -0,0 +1,79 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    _instanceof:
+        Ibexa\Contracts\AutomatedTranslation\Encoder\Field\FieldEncoderInterface:
+            tags: [ 'ibexa.automated_translation.field_encoder' ]
+
+        Ibexa\Contracts\AutomatedTranslation\Encoder\BlockAttribute\BlockAttributeEncoderInterface:
+            tags: [ 'ibexa.automated_translation.block_attribute_encoder' ]
+
+    # field encoder
+    Ibexa\AutomatedTranslation\Encoder\Field\TextLineFieldEncoder: ~
+
+    Ibexa\AutomatedTranslation\Encoder\Field\PageBuilderFieldEncoder:
+        arguments:
+            $blockAttributeEncoderManager: '@Ibexa\AutomatedTranslation\Encoder\BlockAttribute\BlockAttributeEncoderManager'
+            $blockDefinitionFactory: '@Ibexa\FieldTypePage\FieldType\Page\Block\Definition\BlockDefinitionFactory'
+
+    Ibexa\AutomatedTranslation\Encoder\Field\RichTextFieldEncoder:
+        arguments:
+            $richTextEncoder: '@Ibexa\AutomatedTranslation\Encoder\RichText\RichTextEncoder'
+
+    # field encoder manager
+    Ibexa\AutomatedTranslation\Encoder\Field\FieldEncoderManager:
+        arguments:
+            $fieldEncoders: !tagged_iterator 'ibexa.automated_translation.field_encoder'
```

Comment:
quotes can be skipped, especially that there are places in this file where they're not used, so it's a bit inconsistent.